in_toto/models/link.py

```diff
@@ -21,6 +21,10 @@
 import attr
 import json
 from . import common as models__common
+import securesystemslib.formats
+
+FILENAME_FORMAT = "{step_name}.{short_keyid}.link"
+UNFINISHED_FILENAME_FORMAT = ".{step_name}.{short_keyid}.link-unfinished"
```

Comment:
Do you mean something like FILENAME_FORMAT = "{}.{:.8}.link".format(step_name, keyid) ?